src/commontools.c

```diff
@@ -85,7 +92,8 @@ btc_bool hd_gen_master(const btc_chainparams* chain, char* masterkeyhex, size_t
 {
     btc_hdnode node;
     uint8_t seed[32];
-    assert(btc_random_bytes(seed, 32, true));
+    const btc_bool rc = btc_random_bytes(seed, 32, true);
```

Comment:
Note: this fix is also addressed in #142 which is a smaller set of changes.  Rebasing/merging could be helpful.

Reply:
This PR was first so the change should be here.

Reply:
Sure, will do once #142 is merged